### ISSUE TITLE
Remove unstable formatting directives

### ISSUE THREAD

**Issue by romac** [informalsystems/tendermint-rs#464]
As per the output of `cargo fmt` on Rust stable, those are:

- `wrap_comments = true`
- `comment_width = 100`
- `format_strings = false`
- `unstable_features = true`

<details>
<summary>Full output</summary>
<pre>
Warning: can't set `wrap_comments = true`, unstable features are only available in nightly channel.
Warning: can't set `comment_width = 100`, unstable features are only available in nightly channel.
Warning: can't set `format_strings = false`, unstable features are only available in nightly channel.
Warning: can't set `unstable_features = true`, unstable features are only available in nightly channel.
</pre>
</details>

**Comment by xla** [comment]
Comment formatting is quite handy, there is always the option to run this specific command with the `+nightly` toolchain.

**Comment by liamsi** [comment]
ref: https://github.com/informalsystems/tendermint-rs/pull/454#discussion_r456393781
ref: #153 and #121

**Comment by thanethomson** [comment]
Perhaps for now we should recommend in our [contributing guidelines](https://github.com/informalsystems/tendermint-rs/blob/master/CONTRIBUTING.md) that contributors should run `cargo +nightly fmt` prior to pushing their code?

**Comment by romac** [comment]
While I agree that eg. comment formatting is very useful, I'd rather keep the workflow as simple as possible and let contributors submit code formatted with the stable rustfmt.

Aside from that, I wouldn't be suprised if the nightly formatting rules may change from day to day, which makes it more difficult to keep a consistently formatted codebase, with the CI issues that might entail.

I would therefore vote for not relying on nightly-only rustfmt features and bite the bullet until doc comments formatting lands in stable.

What do you think @thanethomson @xla @liamsi?

**Comment by xla** [comment]
Sounds sensible!

**Comment by thanethomson** [comment]
> I would therefore vote for not relying on nightly-only rustfmt features and bite the bullet until doc comments formatting lands in stable.

Makes sense 👍 Let's rather wait for these features to land in stable, and remove any unstable formatting directives for now.

I suppose sometimes you've just gotta be patient and wait until the right day/time to open your 🎁 😁
